src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ModifyRolesForUserTest.java

```diff
@@ -253,6 +253,31 @@ public void ac4_delete_role_existing_user_using_pui_user_manager_for_external_20
         //assertThat(!rolesInfo.contains("pui-organisation-manager"));
     }
 
+    @Test
+    public void ac5_modify_role_existing_user_to_organisation_internal_returns_400_if_org_id_invalid() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String orgIdentifier = (String) response.get("organisationIdentifier");
+        professionalApiClient.updateOrganisation(orgIdentifier, hmctsAdmin);
+
+        IdamOpenIdClient idamOpenIdClient = new IdamOpenIdClient(configProperties);
+        String email = idamOpenIdClient.createUser("pui-organisation-manager");
+        NewUserCreationRequest newUserCreationRequest = professionalApiClient.createNewUserRequest(email);
+        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisation(orgIdentifier, hmctsAdmin, newUserCreationRequest, HttpStatus.CREATED);
+
+        Map<String, Object> searchResponse = professionalApiClient.searchOrganisationUsersByStatusInternal(orgIdentifier, hmctsAdmin, HttpStatus.OK);
+        List<Map> professionalUsersResponses = (List<Map>) searchResponse.get("users");
+        Map professionalUsersResponse = getActiveUser(professionalUsersResponses);
+        String userId = (String) professionalUsersResponse.get("userIdentifier");
+
+        UserProfileUpdatedData userProfileUpdatedData = new UserProfileUpdatedData();
+        RoleName role1 = new RoleName("pui-user-manager");
+        Set<RoleName> roles = new HashSet<>();
+        roles.add(role1);
+        userProfileUpdatedData.setRolesAdd(roles);
+
+        professionalApiClient.modifyUserToExistingUserForPrdAdmin(HttpStatus.BAD_REQUEST, userProfileUpdatedData, "%!*invalid-org-id", userId);

```

Comment:
As per JIRA just put % so that it will try to find all matching orgs.
%!*invalid-org-id means It will try to search. all orgs ids having occurrence of "invalid-org-id"

Reply:
I will use the same organisation identifier: "%7C" that was in the JIRA ticket for the test.

Reply:
I have addressed this in the following commit: https://github.com/hmcts/rd-professional-api/pull/442/commits/532c5871ac1c709b24b0c72fe11d80c318308122